README.md

```diff
@@ -103,65 +104,94 @@ Undefined for boolean options.
   * `inlineValue` { boolean | undefined } Whether option value specified inline,
 like `--foo=bar`.
 * positional tokens
-  * `value` { string } the value of the positional argument in args (i.e. `args[index]`).
+  * `value` { string } The value of the positional argument in args (i.e. `args[index]`).
 * option-terminator token
 
-For example, assuming the following script which uses
-automatic detection of options and no error checking:
+The returned tokens are in the order encountered in the input args. Options that appear
+more than once in args produce a token for each use.
+Short option groups like `-xy` expand to a token for each option. So `-xxx` produces
+three tokens.
+
+For example to use the returned tokens to add support for a negated option like `--no-color`, the tokens
+can be reprocessed to change the value stored for the negated option.
 
 ```mjs
 import { parseArgs } from 'node:util';
-console.log(parseArgs({ strict: false, tokens: true }));
+
+const options = {
+  ['color']: { type: 'boolean' },
+  ['no-color']: { type: 'boolean' },
+  ['logfile']: { type: 'string' },
+  ['no-logfile']: { type: 'boolean' },
+};
+const { values, tokens } = parseArgs({ options, tokens: true });
+
+// Reprocess the option tokens and overwrite the returned values.
+tokens
+  .filter((token) => token.kind === 'option')
+  .forEach((token) => {
+    if (token.name.startsWith('no-')) {
+      // Store foo:false for --no-foo
+      const positiveName = token.name.slice(3);
+      values[positiveName] = false;
+      delete values[token.name];
+    } else {
+      // Resave value so last one wins if both --foo and --no-foo.
+      values[token.name] = token.value ?? true;
+    }
+  });
+
+const color = values.color;
+const logfile = values.logfile ?? 'default.log';
+
+console.log({ logfile, color });
 ```
 
 ```cjs
 const { parseArgs } = require('node:util');
-console.log(parseArgs({ strict: false, tokens: true }));
+
+const options = {
+  ['color']: { type: 'boolean' },
+  ['no-color']: { type: 'boolean' },
+  ['logfile']: { type: 'string' },
+  ['no-logfile']: { type: 'boolean' },
```

Comment:
```suggestion
  color: { type: 'boolean' },
  'no-color': { type: 'boolean' },
  logfile: { type: 'string' },
  'no-logfile': { type: 'boolean' },
```
brackets are only needed for computed properties, and shouldn't be used otherwise.

Reply:
Removed brackets. (Linter wanted consistent quoting.)

Reply:
Consistent quoting is wildly unidiomatic but is fine; using brackets unnecessarily is not.